### DISCUSSION TITLE
Use async.el to not block emacs on large file export

### DISCUSSION THREAD

**Discussion by QiangF** [kjambunathan/org-mode-ox-odt#206] [Ideas]
It's takes a long time for large document, eg. a thesis.

**Comment by kjambunathan** [comment]
> Use async.el to not block emacs on large file export

Use `C-c C-e C-a o O`.  The `C-a` key enables async export.

**Comment by kjambunathan** [comment]
https://github.com/kjambunathan/org-mode-ox-odt/wiki

**Comment by kjambunathan** [comment]
There is async support.   Look for C-a in the export dispatcher menu.
